fem/libceed/ceed.hpp

```diff
@@ -105,6 +118,11 @@ void InitCeedBasisAndRestriction(const FiniteElementSpace &fes,
 /// Return the path to the libCEED q-function headers.
 const std::string &GetCeedPath();
 
+void CeedPAAssemble(const FiniteElementSpace &fes,
```

Comment:
Can we get a couple lines of documentation here?

Reply:
Done, tell me if you like it or if you think I should document more specific points.